engineering/ember.md

```diff
@@ -41,8 +41,4 @@ refactoring. Components are also easier to test.
 
 ### [ember-test-selectors](https://github.com/simplabs/ember-test-selectors)
 
-Import using the following syntax.
-
-```js
-import testSelector from 'ember-test-selectors';
-```
+Use test selectors for any selectors added for testing purposes.
```

Comment:
I'd maybe show an example with `<input [data-test-first-name] />` and

```
// test/pages/form.js
create({
  firstNameField: {
    scope: '[data-test-first-name]'
  }
});
```

Reply:
yea we should definitely add more examples to this for lots of stuff. I just wanted to remove the 1 part that was wrong. For now, I think they can just click the link and see that in the readme